lang/python/python3-unidecode/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-unidecode
+PKG_VERSION:=1.0.23
+PKG_RELEASE:=1
+
+PKG_SOURCE:=Unidecode-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/u/unidecode/
+PKG_HASH:=8b85354be8fd0c0e10adbf0675f6dc2310e56fda43fa8fe049123b6c475e52fb
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/Unidecode-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=GPL-2.0
```

Comment:
A small nitpick: The [readme](https://github.com/avian2/unidecode/blob/master/README.rst#copyright) and [setup.py](https://github.com/avian2/unidecode/blob/master/setup.py#L32) say the project is licensed under GPL v2 or later, so this should be `GPL-2.0+`.

Reply:
Oh... In their Github repository, there's GPL-2.0, that's why it happened. I will change it. Thank you for your review!

Reply:
I don't think GitHub is smart enough to detect the "or later" part :laughing: